docs/.vitepress/config/en.ts

```diff
@@ -291,6 +291,10 @@ function sidebarDeveloperDocs(): DefaultTheme.SidebarItem[] {
           text: "Achievement Templates",
           link: "/developer-docs/achievement-templates",
         },
+        {
+          text: "Save and Password Protection",
+          link: "/developer-docs/save-and-password-protection",
+        },
```

Comment:
Let's also add this to _pt.ts_ and _es.ts_, though commented out. Ideally we keep our navs somewhat in sync across locales.